tests/unittests/Foundation/NSPredicateTests.m

```diff
@@ -324,8 +320,26 @@ - (void)dealloc {
     ASSERT_EQ_MSG(1, [validTestAnimals count], "FAILED: Unable to filter based on class property.");
 }
 
-TEST(NSPredicate, Like) {
-    NSMutableArray* animals = [[[NSMutableArray alloc] init] autorelease];
+TEST(NSPredicate, Matches0) {
```

Comment:
Is it possible to get better names for these tests, since you're touching them?
`NSPredicate, StringsFilteredByRegularExpression`
`NSPredicate, InstanceFieldsFilteredByRegularExpression`